### PR TITLE
docs(sample): normalize sample's region tags

### PR DESCRIPTION
**Pull request by laljikanjareeya**
Make sample's region tags consistent across all the languages, [another PR](https://github.com/firebase/snippets-node/pull/170) for other firestore samples.
//cc @crwilcox

### CODE REVIEW

File: samples/solution-counters.js [hunk 1]
```diff
@@ -16,15 +16,15 @@
 const {Firestore, FieldValue} = require('@google-cloud/firestore');
 
 async function main() {
-  // [START increment_counter]
+  // [START firestore_solution_sharded_counter_increment]
```
Comment (crwilcox) [inline samples/solution-counters.js:19]: similar to the other document, can we keep boht tags for a bit? this aids the transition. 

```
// [START add_lovelace]
// [START firestore_setup_dataset_pt1]


/* SAMPLE CODE */

// [END firestore_setup_dataset_pt1]
// [END add_lovelace]
```

**Review by crwilcox — CHANGES_REQUESTED** [review]

File: samples/solution-counters.js (hunk 1 above)
Comment (laljikanjareeya) [reply samples/solution-counters.js:20]: Done!!

**Comment by tmatsuo** [comment]
@laljikanjareeya 

Let me add some explanation on the snippet-bot comment:
> There are 3 possible violations for not having product prefix.

The bot thinks they are violations because the current data shows the `region_tag_prefix` should be `fs` for Firestore samples. However, it might be possible that the bot is using stale data. 

If you're sure the region tag prefix is `firestore`, just go ahead and merge. I'll appreciate if you file a data quality issue at
https://github.com/googleapis/repo-automation-bots/issues in that case.

Thanks,

**Comment by laljikanjareeya** [comment]
Gentle ping @crwilcox

**Review by crwilcox — APPROVED** [review]

**Pull request merged**